flink-formats/flink-sql-orc/src/main/resources/META-INF/services/NOTICE

```diff
@@ -1,4 +1,4 @@
-flink-orc
```

Comment:
The location of this file is wrong. It needs to be in the META-INF folder, not the services folder.